content/log/explore.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Explore
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+## Overview 
+
+The Log explorer is an optimized view which makes it easy to do any kind of troubleshooting and data exploration.
+
+{{< img src="log/explore/explore_view_with_comments.png" alt="Explore view with comments" >}}
+
+In this view you can:
+
+* [Interact with the Time range](#time-range)
+* [Display logs into lists](#log-list)
+* [Use facets to help you in various investigations](#facets)
+* [Enter search queries](#search-bar)
+
+## Time Range
+Time range allows you to display logs on a given time period. It is symbolised by the timeline directly under the search bar. The timeline can be displayed or wrapped up with the **Show Graph** check box:
+
+{{< img src="log/explore/timeline.png" alt="Timeline" >}}
+
+Change the time periode displayed with quick ranges that are directly available:
+
+{{< img src="log/explore/timerange.png" alt="Timerange" style="width:75%;">}}
+
+## Log list
+The log list is the list of events that match the selected context, a context is defined by a [search bar](#search-bar) filter and a [time range](#time-range).
+{{< img src="log/explore/log_list.png" alt="Log List" >}}
+
+### Sorting the log list
+
+Click on the **date** columns to get your logs in an ascending or descending order in the Log list.
+
+### Filtering the log list
+If you enter a valid query into the [search bar](#search-bar),  words that match your query are highlighted and log displayed match your facet criterias:
+
+Searching for all the logs containing the words "closed connection" from host:i-07a49480a9713bfe6 : 
+{{< img src="log/explore/log_list_highlighted.png" alt="Log List highlighted" >}}
+
+### Displaying a full log
+To display detail of a log line, just click on it:
+
+{{< img src="log/explore/log_in_log_list.png" alt="Log in log list" >}}
+
+### View a log line in context
+When displaying the detail of a log line, ask to « view in context ». 
+It removes any filters from your search bar and display log lines preceding this line and the few log lines following it.
+
+If you send the log lines in syslog format, the hostname and app name corresponding to the log line are automatically be selected.
```

Comment:
> If you send the log lines in syslog format, the hostname and app name corresponding to the log line are automatically be selected.

For syslog-formatted logs, the hostname and app name are automatically selected.

[what does "selected" mean, by the way? ticked in the checkboxes to the left? "selected" is a little unclear.]

Reply:
👍